spec/bourbon/helpers/font_source_declaration_spec.rb

```diff
@@ -26,4 +26,17 @@
       expect(".no-pipeline").to have_rule(rule)
     end
   end
+
+  context "called with local font references" do
+    it "includes local references" do
+      rule = 'src: local("FontA"), ' +
+             'local("Font A"), ' +
+             'url("b.eot?#iefix") format("embedded-opentype"), ' +
+             'url("b.woff2") format("woff2"), ' +
+             'url("b.woff") format("woff"), ' +
+             'url("b.ttf") format("truetype"), ' +
```

Comment:
Use 2 (not 7) spaces for indenting an expression in an assignment spanning multiple lines.